extensions/google-calendar-quickadd/src/utils/oauth.ts

```diff
@@ -0,0 +1,35 @@
+import { auth, calendar_v3 } from "@googleapis/calendar";
+import { OAuthService, withAccessToken } from "@raycast/utils";
+import type { Tool } from "@raycast/api";
+
+// Only request the minimal scopes needed
+const GOOGLE_SCOPES =
+  "https://www.googleapis.com/auth/calendar.events https://www.googleapis.com/auth/calendar.readonly";
```

Comment:
Try to only set `const GOOGLE_SCOPES = "https://www.googleapis.com/auth/calendar";`